src/Microsoft.VisualStudio.Web.CodeGeneration.Tools/project.json

```diff
@@ -26,37 +28,21 @@
     }
   },
   "dependencies": {
-    "Microsoft.DotNet.Cli.Utils": "1.0.0-preview3-*",
-    "Microsoft.Extensions.CommandLineUtils": "1.1.0-*",
-    "Microsoft.Extensions.DotnetToolDispatcher.Sources": {
-      "version": "1.1.0-*",
-      "type": "build"
-    },
-    "Microsoft.Extensions.PlatformAbstractions": "1.1.0-*",
-    "Microsoft.Extensions.FileProviders.Physical": "1.1.0-*",
-    "Microsoft.DotNet.InternalAbstractions": "1.0.0-*",
-    "Microsoft.Extensions.ProjectModel.Abstractions.Sources": {
-      "version": "1.0.0-*",
-      "type": "build"
-    },
-    "Microsoft.Extensions.ProjectModel.MsBuild.Sources": {
-      "version": "1.0.0-*",
-      "type": "build"
-    },
-    "Microsoft.Extensions.ProjectModel.DotNet.Sources": {
-      "version": "1.0.0-*",
-      "type": "build"
-    },
+    "Microsoft.DotNet.Cli.Utils": "1.0.0-preview2-003121",
```

Comment:
The fixes for the ProjectDependencyCommandResolver are in the preview3 version of CLI utils. 
Will need that for msbuild support.